app/build.gradle

```diff
@@ -114,6 +114,11 @@ dependencies {
     implementation 'org.bitcoinj:bitcoinj-tools:0.14.7'
     implementation 'org.java-websocket:Java-WebSocket:1.5.1'
 
+    implementation 'com.github.ktprograms:HaishinKit~kt:b8192ad834'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.0'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.0'
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
```

Comment:
Why this?

Reply:
[HaishinKit](https://github.com/shogo4405/HaishinKit.kt/) is a library for RTMP streaming for Android. I had to make a few changes in order to get it to build with Jitpack, rather than using the version from GitHub packages, which requires an authentication token. The other 3 dependencies are needed but don't seem to be pulled in by HaishinKit.

As I said, this is extremely WIP and I'm just making my current code available.